.env.sample

```diff
@@ -10,7 +10,10 @@ IMAGE_TAG__KROMA_PROVER=v0.1.3
 # L1 RPC endpoint.
 # You can check and use one of the list of RPC endpoints: https://sepolia.dev/
 # Or you can also use Alchemy or Infura, etc.
-L1_RPC_ENDPOINT=https://rpc.sepolia.org
+# You must use a websocket RPC endpoint to run a validator node for below reasons:
+# As an asserter, to respond if a challenge is created to the output you submitted.
+# As a challenger, to observe output submission events and create challenge to an invalid output.
+L1_RPC_ENDPOINT=wss://sepolia.gateway.tenderly.co
```

Comment:
Looks good.
In the future, it would be good to consider distinguishing between the endpoint (JSON-RPC) used by kroma-node and the endpoint (websocket) used by kroma-validator.

Reply:
I applied your suggestion, please check it again :)